ingest/ingest_files.py

```diff
@@ -270,30 +272,11 @@ def open_txt(self, open_file_object, **kwargs):
     def open_pandas(self, file_path, file_type, **kwargs):
         """Opens file as a dataframe """
         open_file_object = kwargs.pop('open_file_object')
-        if file_type == "text/tab-separated-values":
```

Comment:
Worried that changing how we open tsv and csv files will have subtle changes to the behavior of ingest pipeline that our tests won't detect. We should get this change into staging (especially if Sarah is willing to upload some 2.0.0 data into staging) and consider uploading a variety of known good files (csv/txt and also rename to txt) and make sure we don't think this change has unintended side effects.